lib/azure/armrest/storage_account_service.rb

```diff
@@ -95,41 +97,27 @@ def list_all
       #   sas = Azure::Armrest::StorageAccountService(config)
       #
       #   sas.create(
-      #     :name     => "yourstorageaccount1",
-      #     :location => "West US",
-      #     :type     => "Standard_ZRS",
-      #     :tags     => {:YourCompany => true}
+      #     "yourstorageaccount1",
+      #     :location   => "West US",
+      #     :tags       => {:YourCompany => true}
+      #     :properties => {:accountType => "Standard_ZRS"},
       #   )
       #
       # For convenience you may also specify the :resource_group as an option.
       #
-      def create(options = {}, rgroup = armrest_configuration.resource_group)
-        rgroup ||= options[:resource_group]
+      def create(name, options = {}, rgroup = armrest_configuration.resource_group)
+        rgroup = options.delete(:resource_group) || rgroup
```

Comment:
With this code you can do:

    sa.create('foo', {:resource_group => 'x', :something => 'y', :something_else => 'z'})

Instead of:

    sa.create('foo', {:something => 'y', :something_else => 'z'}, 'x')

I just thought the first looked a little nicer and was a little more convenient.